guides/v2.2/magento-functional-testing-framework/release-2/test/annotations.md

```diff
@@ -31,6 +31,7 @@ Recommended use cases of the annotation types:
 - [group] - general functionality grouping.
 - [title] - description of the test purpose.
 - [description] - description of how the test achieves the purpose defined in the title.
+- [skip] - used for tests that are incomplete due to an issue
```

Comment:
changed to:
`- a label for the test to be skipped during generation (for example, an incomplete test blocked by an issue)`